### PR TITLE
Create new database to store user sign up information. Will work on creating login and hashing passwords

### PR DESCRIPTION
**Pull request by robertbernardbrown**

### CODE REVIEW

File: app.py [hunk 1]
```diff
@@ -48,9 +64,18 @@ def choice():
 def final():
 		return render_template('finally.html')
 
-@app.route('/signup')
+@app.route('/signup', methods=['GET', 'POST'])
```
Comment (DanielHabib) [inline app.py:67]: You added 2 new methods here, GET and POST, yet you aren't filtering on either. 
Are you sure you need this?

**Comment by DanielHabib** [comment]
Clean up your description and title also

File: app.py (hunk 1 above)
Comment (robertbernardbrown) [reply app.py]: Yeah in this commit they don't make a lot of sense. I should have left it out for this one. I ended up adding code using those in the subbranch, [https://github.com/robertbernardbrown/Pizza/tree/subpizzalogin2]

Comment (robertbernardbrown) [reply app.py:67]: Yeah in this commit they don't make a lot of sense. I should have left it out for this one. I ended up adding code using those in the subbranch, subpizzalogin2 (Yeah in this commit they don't make a lot of sense. I should have left it out for this one. I ended up adding code using those in the subbranch, subpizzalogin2 (https://github.com/robertbernardbrown/Pizza/tree/subpizzalogin2)

Comment (robertbernardbrown) [reply app.py:67]: I think that's the case anyway, let me know if I'm wrong

**Review by DanielHabib — APPROVED** [review]

**Pull request merged**